tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -177,8 +186,10 @@ if (tensorflow_BUILD_CC_TESTS)
   include(googletest)
 endif()
 
+add_definitions(${ADD_CFLAGS})
```

Comment:
Should these be protected like:
```
if len(ADD_CLAGS) > 0
  add_definitions...
```
I cannot see how add_definitions or link_directories will behave if these are unset.

Reply:
I thought ```add_definitions``` with null strings or unset strings behaves ok (no effect on runtime); however, I'll recheck this issue.